### PR TITLE
Makes shadowling void jaunt fail when on fire

### PR DESCRIPTION
**Pull request by Mqiib**
I'm tired of playing hide and go seek with shadowlings for a FUCKING HOUR when I'm armed to the teeth and they can consistently escape with no way to figure out where they went. It's dumb as hell and absolutely infuriating to play against.

# Wiki Documentation

Note that the jaunt will fail to activate if you're on fire. Let your thralls take the heat!

# Changelog

:cl:  

tweak:  Shadowling void jaunt no longer works while on fire

/:cl:

### CODE REVIEW

**Comment by ghost** [comment]
Shadowling nerfs cringe

**Comment by VaelophisNyx** [comment]
sling nerf, pog

**Comment by Marmio64** [comment]
There are a million counters to slings already, guaranteeing a death sentence when shot by someone once seems unnecessary. Why not instead increase void haunt's stamina drain while on fire so they still can escape if the enemy chooses to not give chase?

**Comment by Hopekz** [comment]
This PR is just kind of introducing a hard counter to slings except they're usually the main antag of the round so if they're taken out as quick as borgs with flashes it probably doesn't make much sense imo

**Comment by Mqiib** [comment]
> There are a million counters to slings already, guaranteeing a death sentence when shot by someone once seems unnecessary. Why not instead increase void haunt's stamina drain while on fire so they still can escape if the enemy chooses to not give chase?

In theory they have a bunch of hard counters, but a lot of those hard counters have been nerfed to the point that they are no longer hard counters (flood lights being extremely easy to turn off and turning off randomly on their own, glowshrooms converting to shadowshrooms). In practice, every other counter to slings is beaten by jaunt. I've had two rounds of dealing with it, it's horribly obnoxious and unfun when they can just disappear at no cost and then the moment they're back in darkness noclip away at meth speed.

**Comment by ilikesaying** [comment]
Woe, molotov be upon ye.

**Comment by Mqiib** [comment]
As per a request by Theos I have removed it going on cooldown after failing.

File: yogstation/code/modules/antagonists/shadowling/shadowling_abilities.dm [hunk 1]
```diff
@@ -751,6 +751,13 @@
 	var/apply_damage = TRUE
 
 /obj/effect/proc_holder/spell/targeted/void_jaunt/cast(list/targets,mob/living/user = usr)
+	if(istype(usr, /mob/living/carbon))	//If we're not an ascendant sling
```
Comment (tattax) [inline yogstation/code/modules/antagonists/shadowling/shadowling_abilities.dm]:
```suggestion
	if(iscarbon(usr))	//If we're not an ascendant sling
```

**Review by JamieD1 — APPROVED** [review]
Completely Agree

**Comment by ghost** [comment]
Fuck

**Pull request merged**